src/pages/nonprofit/issues/[id].tsx

```diff
@@ -1,9 +1,218 @@
-import { Flex } from "@chakra-ui/react";
+import {
+  Button,
+  Flex,
+  VStack,
+  Text,
+  HStack,
+  Image,
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalBody,
+  useDisclosure,
+} from "@chakra-ui/react";
+import Link from "next/link";
+import { useRouter } from "next/router";
+import React, { useEffect, useState } from "react";
+import { HiOutlineChevronLeft } from "react-icons/hi";
+import { nonprofitGetIssue, nonprofitUpdateIssue } from "src/actions/Issue";
+import { nonprofitGetUser } from "src/actions/User";
+import LoadingIndicator from "src/components/shared/LoadingIndicator";
+import { dateToMMDDYYYY } from "src/utils/dates";
+import { showError } from "src/utils/notifications";
+import { Issue, IssueStatus, MaintenanceType, User } from "src/utils/types";
+import urls from "src/utils/urls";
 
 function NonprofitIssueViewPage() {
+  const [issue, setIssue] = useState<Issue>();
+  const router = useRouter();
+  const [projectId, setProjectId] = useState("617ea71c069ce109e2ae9f83");
+  const [isLoading, setLoading] = useState(true);
+  const [hasReviewer, setHasReviwer] = useState(false);
+  const { isOpen, onOpen, onClose } = useDisclosure();
+
+  const closeIssue = async () => {
+    const issueId = router.query.id as string;
+    const issueClosed = await nonprofitUpdateIssue(issueId, projectId, {
```

Comment:
Remember that frontend actions can throw errors, so they should always be wrapped in a try catch block.